sklearn/compose/_column_transformer.py

```diff
@@ -141,6 +141,12 @@ class ColumnTransformer(TransformerMixin, _BaseComposition):
 
         .. versionadded:: 1.0
 
+    n_features_in_ : int
+        Number of features seen during :term:`fit`. Only defined if the
+        underlying transformers expose such an attribute when fit.
+
+        .. versionadded:: 0.24
```

Comment:
If this is not covered by common tests, maybe we should add a dedicated test.

Reply:
There are many untested estimators. I'd rather try to find a way to include more estimators in the common tests. We've already started to discuss that with @glemaitre.